index.html

```diff
@@ -0,0 +1,71 @@
+<html>
+
+<head>
+    <title>Nikos Village Websitelol</title>
+</head>
+
+<body>
+    
+    <div class="FadeIn"><h1>Welcome to Drosia!</h1></div>
+  
+    <div class="Fadein"><p>This is the test paragraph for the site. Maybe one day
+        it wont look like it was made in 1999.
+    </p></div>
+
+    <div  class="FadeIn"><button>Test Button</button></div>
```

Comment:
You can attach classes directly to other tags as well, so we can remove the need for a div here
```suggestion
    <button class="FadeIn">Test Button</button>
```